cmd/root_test.go

```diff
@@ -27,6 +29,19 @@ type globalTestState struct {
 	expectedExitCode int
 }
 
+func getFreePort(t *testing.T) int {
+	freePortAddr, err := net.ResolveTCPAddr("tcp", "localhost:0")
```

Comment:
Tiny nit, but for better compatibility, you could consider using [`JoinHostPort`](https://pkg.go.dev/net#JoinHostPort) here 👍🏻

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/a6823b208ee3353e499354e2fc5f2fe6dcb61dc8..b37b0dd69a8f5e11806e202e001f2d490452e5a9), discussion in https://github.com/grafana/k6/pull/2789#discussion_r1037105698